i18n/src/main/resources/openfire_i18n_es.properties

```diff
@@ -2674,13 +3159,63 @@ client.connections.settings.ping.enable=Enviar una solicitud XMPP Ping a los cli
 client.connections.settings.ping.disable=No enviar una solicitud XMPP Ping a los clientes inactivos.
 
 # Connection type and mode
-connection-type.socket-s2s=server-to-server (federation)
-connection-type.socket-c2s=client-to-server
+connection-type.socket-s2s=servidor-a-servidor (federación)
+connection-type.socket-c2s=cliente-a-servidor
 connection-type.bosh-c2s=HTTP-binding (BOSH)
-connection-type.webadmin=admin console
-connection-type.component=external component
-connection-type.connection-manager=connection manager
-connection-type.unspecified=unspecified
-connection-mode.plain=plain text (with STARTSSL)
-connection-mode.legacy=encrypted (legacy-mode)
-connection-mode.unspecified=unspecified
+connection-type.webadmin=consola de administración
+connection-type.component=componente externo
+connection-type.connection-manager=administrador de conexiones
+connection-type.unspecified=no especificado
+connection-mode.plain=texto plano (con STARTSSL)
+connection-mode.legacy=cifrado (legacy-mode)
+connection-mode.unspecified=no especificado
+
+# DNS SRV Records check
```

Comment:
Everything under this line is in English - although it doesn't hurt to have this in, I'm assuming that the intention was to translate this into Spanish too?

Reply:
Indeed, haven't noticed this. Maybe this is kind of a placeholder for future updates. Or a workaround for an issue where Openfire showed ??? instead of English if strings were missing in translation. Though, i think this issue is already fixed.

Reply:
I did translate that. Maybe I messed up with the files when I commited. I'll check later.